packages/commerce/backend/src/open-payments/service.ts

```diff
@@ -155,9 +153,85 @@ export class OpenPayments implements IOpenPayments {
       throw new InternalServerError()
     })
 
+    await order.$query().patch({
+      quoteId: quote.id,
+      paymentPointerUrl: customerPaymentPointer.id,
+      continueToken: outgoingPaymentGrant.continue.access_token.value,
+      continueUri: outgoingPaymentGrant.continue.uri
+        .replace('localhost', 'rafiki-auth')
+        .replace('auth/', '') // TODO: Remove this when upgrading to the new version.
+    })
+
     return outgoingPaymentGrant
   }
 
+  public async createOutgoingPayment(
+    order: Order,
+    interactRef: string
+  ): Promise<void> {
+    try {
+      if (!order.paymentPointerUrl) {
+        this.logger.error("Order is missing customer's payment pointer.")
+        throw new InternalServerError()
+      }
+
+      if (!interactRef) {
+        this.logger.error('Missing "interactRef".')
+        throw new InternalServerError()
+      }
+
+      if (!order.continueToken || !order.continueUri) {
+        this.logger.debug(`continueToken - ${order.continueToken}`)
+        this.logger.debug(`continueUri - ${order.continueUri}`)
+        this.logger.error('Could not find order continuation token or URI.')
+        throw new InternalServerError()
+      }
+
+      const continuation = await this.opClient.grant
+        .continue(
+          {
+            accessToken: order.continueToken,
+            url: order.continueUri
+          },
+          {
+            interact_ref: interactRef
+          }
+        )
+        .catch((err) => {
+          this.logger.error('Could not finish the continuation request.')
+          this.logger.error(err)
+          throw new InternalServerError()
+        })
+
+      await this.opClient.outgoingPayment
+        .create(
+          {
+            paymentPointer: order.paymentPointerUrl,
+            accessToken: continuation.access_token.value
+          },
+          {
+            quoteId: order.quoteId,
+            metadata: {
+              orderRef: order.id
+            }
+          }
+        )
+        .catch(() => {
```

Comment:
log the error as well

Reply:
Open Payments already has a logger implemented which will automatically log the error. I'll need to remove `console.log(err)` from `catch` blocks.